frontend/src/test/scala/bloop/bsp/BspBaseSuite.scala

```diff
@@ -340,7 +340,7 @@ abstract class BspBaseSuite extends BaseSuite with BspClientTest {
   def createBspCommand(configDir: AbsolutePath): Commands.ValidatedBsp = {
     protocol match {
       case BspProtocol.Tcp =>
-        val portNumber = 5202 + scala.util.Random.nextInt(2000)
+        val portNumber = 7001 + scala.util.Random.nextInt(20000)
```

Comment:
Would it be possible to find an open port using `portNumber=0`?

Reply:
We're testing the bsp server as a blackbox, if we pick 0 here we don't know what port will the server pick from the client side and therefore cannot materialise the connection.